tokio-util/src/cfg.rs

```diff
@@ -0,0 +1,19 @@
+macro_rules! cfg_codec {
```

Comment:
Why does `tokio-util` need these helpers when none of the other crates (afaict) do?

Reply:
`tokio` needs the helpers too?

Reply:
TBH, tokio-util is kind of under developed ATM. We may want it to be tokio-codec :shrug: